instagram/src/Components/authenticate/authenticate.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Login from '../Login/Login.js'
+import PostPage from '../PostContainer/PostsPage'
+
+const Authenticate = App => {
+  return class extends React.Component {
+    constructor(){
+      super()
+      this.state = {
+        login: false
+      }
+    }
+
+    changeLogIn = e => {
+    	const loggedIn = {login: true}
+    	this.setState({ login: loggedIn })
+    }
```

Comment:
Awesome work here! Instead of creating a new variable here, could you just set login to true?